Core/Server/Request.php

```diff
@@ -58,61 +58,57 @@ private function _setup() {
 
             foreach ($_SERVER as $name => $value)
             {
+                if (strpos($name, 'HTTP_') === 0) {
+                    $name = strtolower(str_replace('_', '-', substr($name, 5)));
 
-                if (strpos($name,'HTTP_') !== false) {
```

Comment:
Every tag which came from outer network will be marked with  `HTTP_` marker, and headers which standardized for any requests coming without this marker. It's totally accurate having boolean comparison in this scenario.